public/static/docs/user-guide/analytics.md

```diff
@@ -25,12 +25,11 @@ User and event data have a 14 month retention period.
 
 DVC's analytics record the following information per event:
 
-- The DVC version, e.g. `0.22.0`
-- The operating system information, e.g. `linux`, `ubuntu`, `14.04`, etc
-- The underlying version control system, e.g. `git`
+- The DVC version, e.g. `0.8.0`
+- The operating system information, e.g. `linux`, `ubuntu`, `14.04`, etc.
 - Command type, e.g. `CmdDataPull`
 - Command return code, e.g. `1`
-- Way the DVC was installed, e.g. `binary`
+- Way the DVC was installed, e.g. `Binary`
```

Comment:
are you sure we send it like this?

Reply:
No, you're right. I was thinking of the `dvc version` output I guess. Change undone.